addons/hr_holidays/models/hr_leave.py

```diff
@@ -682,12 +683,8 @@ def _check_date(self):
                                                     state=conflicting_holiday_data['state'])
                     conflicting_holidays_strings.append(conflicting_holidays_string)
                 conflicting_employees = set(employee_ids) - set(conflicting_holidays.employee_id.ids)
-                # Only one employee has a conflicting holiday
-                if len(conflicting_employees) == len(employee_ids) - 1:
-                    raise ValidationError(_('You can not set two time off that overlap on the same day for the same employee.\nExisting time off:\n%s') %
-                                          ('\n'.join(conflicting_holidays_strings)))
-                raise ValidationError(_('You can not set two time off that overlap on the same day for the same employees.\nExisting time off:\n%s') %
-                                      ('\n'.join(conflicting_holidays_strings)))
+                raise ValidationError(_("An employee already booked time offs which overlap with this period:" +
```

Comment:
Correct me if I'm wrong but "time offs" sounds weird to me, I don't think it's very natural to pluralize it since "time" is indefinite. It's not "a time", it's just time. I book time off, not "a time off". So I would just write "An employee already booked time off which overlaps with this period".
Can a native English speaker back me off? @edi-odoo ?

Reply:
You’re completely correct @Zinston 👍

Reply:
But looking at the diff, it seems this message shouldn’t say “You” instead of “An employee”.
No?

Reply:
If we want use "You" instead of "An employee" then It should be like
`You already booked time off for this employee which overlap with this period`
as user creatimg time of for other employees :thinking: 

@jija2 What do you think?

Reply:
The thing is that it might have been booked by the employee as well as the administrator so we cannot state it for sure.
Let's keep "An employee already booked time off.." which will remain correct in any case